package.json

```diff
@@ -51,7 +51,7 @@
         "lint-staged": "^10.1.1",
         "prettier": "^2.0.2",
         "prettier-config-landr": "^0.0.7",
-        "ts-jest": "^25.3.1",
+        "ts-jest": "^25.4.0",
```

Comment:
Fixes a ts-jest issue where ts-jest would fail to find the test file after changes have been made to it.